libs/designsystem/src/lib/components/form-field/_form-field-inputs.shared.scss

```diff
@@ -1,9 +1,5 @@
 @use '~@kirbydesign/core/src/scss/utils';
-
-$form-field-input-font-family: var(--kirby-font-family);
-$form-field-input-line-height: 1.5;
-$form-field-input-padding: 1em;
-$form-field-label-height: 24px;
+@import '~@kirbydesign/core/src/scss/base/variables';
```

Comment:
We use Sass Modules, so `@import` should not be used.

See e.g. https://sass-lang.com/documentation/at-rules/use and https://css-tricks.com/introducing-sass-modules/